compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/util/StubGeneratorExtensions.kt

```diff
@@ -26,6 +26,8 @@ open class StubGeneratorExtensions {
         stubGenerator: DeclarationStubGenerator,
     ): IrClass? = null
 
+    open fun getContainerSource(descriptor: DeclarationDescriptor): DeserializedContainerSource? = null
```

Comment:
Is this method needed now, or can its usage be reverted to how it was before?

Reply:
Makes perfect sense!

Done.